test/sdf_test.cpp

```diff
@@ -96,204 +94,141 @@ TEST(SDF, BatchValidation) {
-#endif
-
-  EXPECT_EQ(cubeVoid.Status(), Manifold::Error::NoError);
-  EXPECT_EQ(cube.Genus(), 0);
-  auto prop = cube.GetProperties();
-  EXPECT_NEAR(prop.volume, 8, 0.001);
-  EXPECT_NEAR(prop.surfaceArea, 24, 0.001);
-  EXPECT_NEAR(bounds.min.x, -1, precision);
-  EXPECT_NEAR(bounds.min.y, -1, precision);
-  EXPECT_NEAR(bounds.min.z, -1, precision);
-  EXPECT_NEAR(bounds.max.x, 1, precision);
-  EXPECT_NEAR(bounds.max.y, 1, precision);
-  EXPECT_NEAR(bounds.max.z, 1, precision);
-}
-
-TEST(SDF, BatchResize) {
-  const float size = 20;
-  Manifold layers(LevelSetBatch(
-      [](std::vector<glm::vec3> pvec) {
-        std::vector<float> dvec(pvec.size());
-        for (int i = 0; i < pvec.size(); i++) {
-          dvec[i] = Layers()(pvec[i]);
-        }
-        return dvec;
-      },
-      {glm::vec3(0), glm::vec3(size)}, 1));
-#ifdef MANIFOLD_EXPORT
-  if (options.exportModels) ExportMesh("layers.gltf", layers.GetMesh(), {});
-#endif
-
-  EXPECT_EQ(layers.Status(), Manifold::Error::NoError);
-  EXPECT_EQ(layers.Genus(), -8);
-}
-
-TEST(SDF, BatchSineSurface) {
-  Mesh surface(LevelSetBatch(
-      [](std::vector<glm::vec3> pvec) {
-        std::vector<float> dvec(pvec.size());
-        for (int i = 0; i < pvec.size(); i++) {
-          float mid = glm::sin(pvec[i].x) + glm::sin(pvec[i].y);
-          dvec[i] = (pvec[i].z > mid - 0.5 && pvec[i].z < mid + 0.5) ? 1 : 0;
-        }
-        return dvec;
-      },
-      {glm::vec3(-4 * glm::pi<float>()), glm::vec3(4 * glm::pi<float>())}, 1));
-  Manifold smoothed = Manifold::Smooth(surface).Refine(2);
-
-  EXPECT_EQ(smoothed.Status(), Manifold::Error::NoError);
-  EXPECT_EQ(smoothed.Genus(), -2);
-
-#ifdef MANIFOLD_EXPORT
-  if (options.exportModels)
-    ExportMesh("batchsinesurface.glb", smoothed.GetMeshGL(), {});
-#endif
+  for (const Box bound : {bounds, boundsBatch}) {
+    EXPECT_NEAR(bounds.min.x, -outerBound, precision);
+    EXPECT_NEAR(bounds.min.y, -outerBound, precision);
+    EXPECT_NEAR(bounds.min.z, -outerBound, precision);
+    EXPECT_NEAR(bounds.max.x, outerBound, precision);
+    EXPECT_NEAR(bounds.max.y, outerBound, precision);
+    EXPECT_NEAR(bounds.max.z, outerBound, precision);
+  }
 }
 
 TEST(SDF, Bounds) {
   const float size = 4;
   const float edgeLength = 0.5;
-
-  Mesh levelSet = LevelSet(
-      CubeVoid(), {glm::vec3(-size / 2), glm::vec3(size / 2)}, edgeLength);
-  Manifold cubeVoid(levelSet);
-  Box bounds = cubeVoid.BoundingBox();
-  const float precision = cubeVoid.Precision();
+  for (Mesh levelSet :
+       {LevelSet(CubeVoid(), {glm::vec3(-size / 2), glm::vec3(size / 2)},
+                 edgeLength),
+        LevelSetBatch(
+            [](std::vector<glm::vec3> pvec) {
+              std::vector<float> dvec(pvec.size());
+              for (int i = 0; i < pvec.size(); i++) {
+                dvec[i] = CubeVoid()(pvec[i]);
+              }
+              return dvec;
+            },
+            {glm::vec3(-size / 2), glm::vec3(size / 2)}, edgeLength)}) {
```

Comment:
Better, but I think we can still reduce a bunch more duplication in these examples by making a helper like `std::pair<Mesh, Mesh> LevelSet2(func, bounds, edgeLength)`.